### ISSUE TITLE
TypeError: 'file' is an invalid keyword argument for open()

### ISSUE THREAD

**Issue by andrewchenshx** [Nuitka/Nuitka#836]
* Nuitka version, full Python version and Platform (Windows, OSX, Linux ...)

0.6.8.4
Python: 3.7.4 (default, Aug  9 2019, 18:34:13) [MSC v.1915 64 bit (AMD64)]
Executable: C:\Users\hp\Anaconda3\python.exe
OS: Windows
Arch: x86_64

* How did you install Nuitka and Python (pip, anaconda, deb, rpm, from source,
Nuitka:pip
Python:anaconda

* If possible please supply a [Short, Self Contained, Correct, Example](http://sscce.org/)
  that demonstrates the issue i.e a small piece of code which reproduces
  the issue and can be run with out any other (or as few as possible)
  external dependencies.

```
import click


@click.group()
def cli():
    pass


@click.command()
@click.option('--filename', prompt=True, help='file Name')
def test1(filename):
    with open(file=filename, mode='w') as file:
        file.write('ok')

@click.command()
@click.option('--filename', prompt=True, help='file Name')
def test2(filename):
    with open(filename, 'w') as file:
        file.write('ok')


cli.add_command(test1)
cli.add_command(test2)

if __name__ == '__main__':
    cli()

```

build command:
nuitka --standalone --mingw64 --show-memory --show-progress  --include-package=impo
rtlib --recurse-all --output-dir=bin_open  test_open.py

execute command
test_open test1 --filename 'd:\file'        #   TypeError: 'file' is an invalid keyword argument for open()
test_open test2 --filename 'd:\file'        # works

**Comment by kayhayen** [comment]
It seems, we didn't properly have the first argument for the open built-in defined. It was still what it was for Python2 in Nuitka for Python3. Expect a hotfix with this soonish.

**Comment by kayhayen** [comment]
Seems I messed this one up, and it didn't make it to the hotfix, probably due to me forgetting to put the factory label. Anyway, it's on develop now.

**Comment by kayhayen** [comment]
Released as part of 0.6.10
